webanno/webanno
# #1390 - Move focus to feature name field when creating new feature

**What's in the PR**
- Use AJAX to load the feature detail panel
- Try putting the focus on the name field (which should work but for some reason has no effect so far...)

**How to test manually**
* Create a new feature
* See if the focus goes to the feature name input field

**Automatic testing**

**Documentation**
